### ISSUE TITLE
GeoCompJulia

### ISSUE THREAD

**Issue by felixcremer** [JuliaGeo/JuliaGeoTutorials#2]
We discussed during the JuliaGeo Community session at JuliaCon24 that we would like to have a Geocompx version in Julia. I am not sure, whether this should end up living here, but it seemed to be the most natural place to discuss such efforts. 

 Should we just fork the geocompy repository and start translating every notebook?
I don't see clearly what exactly is part of the setup that we would need to copy but that might be a start. 
 We could also only copy the notebooks and make the setup ourselves from a plain quarto book.

**Comment by asinghvi17** [comment]
Do we want a triple language thing via e.g. Quarto?  We can use the [tabset](https://quarto.org/docs/interactive/layout.html#tabset-panel) features, and this would probably let us rope in the SDSL guys also.

**Comment by felixcremer** [comment]
Yes, that would be really nice, but I would separate that into two tasks. One is the generation of the Julia code, and the other is getting a good tooling for running the three languages in the same quarto notebook. 

It would still be good to have the content in a separate julia book.

**Comment by asinghvi17** [comment]
@evetion this is also your comment in #3 I think.

**Comment by asinghvi17** [comment]
Yeah I agree, we can start the Julia book first.  I'm not too familiar with Quarto but it looks pretty translateable...and a lot of the setup there seems to be extra cached files which we can just skip tbh, since ideally we will generate the entire book in CI.

**Comment by felixcremer** [comment]
The geocomp julia book efforts are now living here: https://github.com/geocompx/geocompjl
